twake/backend/node/src/services/channels/entities/channel.ts

```diff
@@ -8,13 +9,18 @@ export enum VisibilityEnum {
 
 export class Channel {
   // uuid-v4
+  @Type(() => String)
   company_id: string;
 
   // "uuid-v4" | "direct"
+  @Type(() => String)
   workspace_id: string;
 
+  @Type(() => String)
+  @Exclude({ toPlainOnly: true })
```

Comment:
Do you need to declare _id in the entities as this is something
- that must not be sent to the frontend
- that does not exists in cassandra
?

Reply:
I need to check how to handle it in the schema, in mongo and in cassandra to only use `id`.
This will be part of some other PR.

Reply:
OK got it, pushing something within the next minutes 🍻

Reply:
Here it is 🍺